azerothcore/azerothcore-wotlk
# fix(Core/Ulduar): Hodir flash freeze safe spots

## CHANGES PROPOSED:
Hodir now creates 2 safe spots for Flash Freeze in 10-man and 3 safe spots in 25-man.

## ISSUES ADDRESSED:
Closes #3556 

## HOW TO TEST THE CHANGES:
Enter 10-man Ulduar with 3 characters.
.go xyz 2000.61 -245.43 432.69 603
Hodir should create 2 safe spots for flash freeze.

Enter 25-man Ulduar with 3 characters.
.go xyz 2000.61 -245.43 432.69 603
Hodir should create 3 safe spots for flash freeze.


## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
